central/jwt/jwt.go

```diff
@@ -51,17 +54,34 @@ func GetPrivateKeyBytes() ([]byte, error) {
 }
 
 func create() (tokens.IssuerFactory, tokens.Validator, error) {
+	// Load initial key so that we would immediately fail in case
+	// it's not present.
+	initialPrivateKey, err := loadPrivateKey(privateKeyDir)
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "loading initial value of JWT private key")
+	}
+	privateKeyStore := jwt.NewSinglePrivateKeyStore(initialPrivateKey, keyID)
+	publicKeyStore := jwt.NewDerivedPublicKeyStore(privateKeyStore, keyID)
+	jwt.WatchPrivateKeyDir(privateKeyDir, loadPrivateKey, func(key *rsa.PrivateKey) {
+		privateKeyStore.UpdateKey(keyID, key)
+	})
+
+	issuerFactory, validator := tokens.CreateIssuerFactoryAndValidator(issuerID, privateKeyStore, publicKeyStore, keyID)
+	return issuerFactory, validator, nil
+}
+
+// We pass parameter here to satisfy WatchPrivateKeyDir interface.
+func loadPrivateKey(_ string) (*rsa.PrivateKey, error) {
```

Comment:
nit: to potentially circumvent adding the parameter here, we may do the same inline in l.65:
```go
	jwt.WatchPrivateKeyDir(privateKeyDir, func(_ string) (*rsa.PrivateKey, error) { return loadPrivateKey() }, func(key *rsa.PrivateKey) {
		privateKeyStore.UpdateKey(keyID, key)
	})
```

Feel like this is more clean instead of passing `privateKeyDir` to `loadPrivateKey` in l.59.